projects/driver_controls/src/stm32f0xx/ads1015.c

```diff
@@ -163,6 +163,9 @@ StatusCode ads1015_read_raw(Ads1015Storage *storage, Ads1015Channel channel, int
     return status_code(STATUS_CODE_INVALID_ARGS);
   }
   *reading = storage->channel_readings[channel];
+  if (!channel_is_enabled(storage, channel)) {
+    return STATUS_CODE_UNREACHABLE;
```

Comment:
`status_code`

I don't think unreachable is the right error - I think it's more of an invalid argument, as reading a disabled channel is invalid.